src/test/regress/sql/multi_extension.sql

```diff
@@ -44,6 +55,7 @@ ALTER EXTENSION citus UPDATE TO '6.0-10';
 ALTER EXTENSION citus UPDATE TO '6.0-11';
 ALTER EXTENSION citus UPDATE TO '6.0-12';
 ALTER EXTENSION citus UPDATE TO '6.0-13';
+ALTER EXTENSION citus UPDATE TO '6.0-14';
 
```

Comment:
We should do the schema check after the upgrade scripts as well, that way we check whether they individually also do the correct thing, or whether they rely on earlier state.